stack.yaml.lock

```diff
@@ -0,0 +1,19 @@
+# This file was autogenerated by Stack.
```

Comment:
Accidentally committed this file?

Reply:
wasn't sure whether to include it or not,
I've removed it now

Reply:
Well I don't use stack much so I am not sure what the purpose of the lock file is and whether it should be committed. @psibi would know I guess.